lib/widgets/story/story_page.dart

```diff
@@ -180,7 +182,7 @@ class StoryPageState extends State<StoryPage> {
                               }
                             },
                             child: FlareActor(
-                              'assets/chimp_ik.flr',
+                              'assets/chimp_iktest.flr',
```

Comment:
there is already one in assets/character/chimp_ik.flr. Remove these assets. Use that one